src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -182,16 +181,20 @@ protected void runTool( final JavaSparkContext ctx ) {
         final HopscotchUniqueMultiMap<String, Integer, QNameAndInterval> qNamesMultiMap = intervalsAndQNameMap._2;
 
         // supplement the template names with other reads that share kmers
-        final List<Tuple2<Integer, String>> intervalDispositions;
-        if ( intervalOnlyAssembly ) intervalDispositions = new ArrayList<>();
-        else intervalDispositions = addAssemblyQNames(params, ctx, kmersToIgnoreFile, qNamesMultiMap, intervals.size(),
-                                                        allPrimaryLines, locations, pipelineOptions);
+        final Map<Integer, String> intervalDispositions;
+        if ( intervalOnlyAssembly ) {
+            intervalDispositions = new HashMap<>();
+        }
```

Comment:
Ugh, I hate to be that guy, but can you put this all one line? 

`} else {` 

I believe that is our coding style standard.